addon/components/discover-page/component.js

```diff
@@ -472,11 +472,22 @@ export default Ember.Component.extend(Analytics, hostAppName, {
             if (!filterList.length || (key === 'providers' && this.get('theme.isProvider')))
                 return;
 
-            filters.push({
-                terms: {
-                    [val]: filterList
+            if (val == 'subjects') {
```

Comment:
I might be missing something, but this looks like it would change the behavior not only on 'subjects', but if the filter type is something else, it would not push to 'filters'. Shouldn't the original `filters.push({terms: {[val]: filterList })` be on an else? Or is the behavior for things like provider filter handled elsewhere?